### PR TITLE
[RUM-6371] Micro frontend support documentation

### PR DESCRIPTION
**Pull request by thomas-lebeau**
### What does this PR do? What is the motivation?

Add documentation about the support of micro frontends using the breforeSend strategy

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 1]
```diff
@@ -370,9 +370,9 @@ The potential `context` values are:
 | RUM event type   | Context                   |
 |------------------|---------------------------|
 | View             | [Location][6]                  |
-| Action           | [Event][7]                     |
-| Resource (XHR)   | [XMLHttpRequest][8] and [PerformanceResourceTiming][9]            |
-| Resource (Fetch) | [Request][10], [Response][11], and [PerformanceResourceTiming][9]      |
+| Action           | [Event][7] and handling stack                     |,
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:373]:
```suggestion
| Action           | [Event][7] and handling stack                     |
```

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 2]
```diff
@@ -1162,6 +1164,91 @@ However, this feature comes with some **limitations**:
 - The feature is incompatible with the `trackSessionAcrossSubdomains` options because `localStorage` data is only shared among the same origin (login.site.com ≠ app.site.com)
 - `localStorage` is limited to 5 MiB by origin, so the application-specific data, Datadog contexts, and other third-party data stored in local storage must be within this limit to avoid any issues
 
+## Micro frontend
+
+Starting with version 5.22, the RUM Browser SDK supports micro frontend architectures. The mechanism is based on stacktrace, to use it, you must be able to extract service and version properties from your application's file paths and filenames.
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:1169]:
```suggestion
Starting with version 5.22, the RUM Browser SDK supports micro frontend architectures. The mechanism is based on stacktrace. To use it, you must be able to extract service and version properties from your application's file paths and filenames.
```

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 3]
```diff
@@ -1162,6 +1164,91 @@ However, this feature comes with some **limitations**:
...
+
+### How to use it
+
+In the beforeSend property, you can override the service and version properties. To help you identify where the event originated, a new property `context.handlingStack` was introduced.
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:1173]:
```suggestion
In the `beforeSend` property, you can override the service and version properties. To help you identify where the event originated, use the `context.handlingStack` property.
```

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 4]
```diff
@@ -370,9 +370,9 @@ The potential `context` values are:
 | RUM event type   | Context                   |
 |------------------|---------------------------|
 | View             | [Location][6]                  |
-| Action           | [Event][7]                     |
-| Resource (XHR)   | [XMLHttpRequest][8] and [PerformanceResourceTiming][9]            |
-| Resource (Fetch) | [Request][10], [Response][11], and [PerformanceResourceTiming][9]      |
+| Action           | [Event][7] and handling stack                     |,
+| Resource (XHR)   | [XMLHttpRequest][8] [PerformanceResourceTiming][9], and handling stack            |
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:374]:
```suggestion
| Resource (XHR)   | [XMLHttpRequest][8], [PerformanceResourceTiming][9], and handling stack            |
```

**Review by hestonhoffman — COMMENTED** [review]
Couple small edits

**Review by cy-moi — APPROVED** [review]

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**